app/models/miq_action.rb

```diff
@@ -848,6 +848,42 @@ def action_container_image_analyze(action, rec, inputs)
     rec.scan
   end
 
+  def action_container_image_prevent(action, rec, inputs)
+    unless rec.kind_of?(ContainerImage)
+      MiqPolicy.logger.error("MIQ(#{__method__}): Unable to perform action [#{action.description}],"\
+                             " object [#{rec.inspect}] is not a Container Image")
+      return
+    end
+
+    unless rec.try(:ext_management_system).kind_of?(ManageIQ::Providers::Openshift::ContainerManagerMixin)
+      MiqPolicy.logger.error("MIQ(#{__method__}): Unable to perform action [#{action.description}], "\
+                             "only applicable for OpenShift Providers")
+      return
+    end
+
+    unless rec.digest.present?
+      MiqPolicy.logger.error("MIQ(#{__method__}): Unable to perform action [#{action.description}], "\
+                             "ContainerImage is not linked with an openshift image")
```

Comment:
Should be consistent with case.  I think it should be `OpenShift` to match other logging.